data/events/scripts/player.lua

```diff
@@ -273,3 +273,26 @@ function Player:onGainSkillTries(skill, tries)
 	end
 	return tries * configManager.getNumber(configKeys.RATE_SKILL)
 end
+
+function Player:onWrapItem(item)
+	local tile = Tile(item:getPosition())
+	if tile and not tile:getHouse() then
```

Comment:
This should be `not tile or not tile:getHouse()`

Reply:
Currently, if the tile does not exist (for some reason) it will skip this block and might execute `item:moveTo(tile)`.